tests/ExpectedOutputTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+
+final class ExpectedOutputTest extends TestCase {
+    public function testCanOutputExpected(): void{
+        $expectedOutput = "Starbucks Rio De Janeiro,134.7088".PHP_EOL.
+            "Starbucks Sydney,136.2776".PHP_EOL.
+            "Starbucks Moscow,178.4328".PHP_EOL;
+
+        $this->expectOutputString($expectedOutput);
+
+        $coordX = "47.6";
+        $coordY = "-122.4";
+        $csvFile = "coffee_shops.csv";
+
+        $v = new GetMyCoffee($coordX, $coordY, $csvFile);
+        $arr = $v->getNearShops();
+        CoffeeShop\PrintMyCoffee::console($arr);
```

Comment:
Is a good integretion test, not sure what is the benefit of asserting on the console output vs on asserting on the result array.

Reply:
Both unit test and integration tests should be done.
Asserting on the console ensures that the array is well interpreted and printed especially when encoding is taken into consideration.